BattleSystem.js

```diff
@@ -119,13 +119,9 @@ class Timeline {
         // Copy current battle state to next node.
         nextNode.copy(node);
 
-        // Apply player actions
-        var results = {
-            playerActionsResults: [],
-            enemyActionResult: undefined
-        };
+        node.characterActionResults = [];
```

Comment:
Each `TimelineNode` now has a `characterActionResults` array and a `enemyActionResult` object. These are the results returned by the `ActionExecutor`. They have some metadata that can be parsed, but most importantly they will have `executed` as `false` if the action fails. We can use this to display to the player when one of their actions is broken, such as trying to use an item the character no longer has.

Reply:
Perfect! Thanks! We may not get to adding a place for it in the UI tonight due to time, but I'll try my best.